### PR TITLE
Fix several pylint warning in debugger python client

### PR DESCRIPTION
**Pull request by polaroi8d**
JerryScript-DCO-1.0-Signed-off-by: Levente Orban orbanl@inf.u-szeged.hu

### CODE REVIEW

File: jerry-debugger/jerry-client-ws.py [hunk 1]
```diff
@@ -169,6 +172,13 @@ def insert_breakpoint(self, args):
         else:
             set_breakpoint(self.debugger, args)
 
+    def disable_args(self, args):
+        if args != "":
+            print("Error: No argument expected")
```
Comment (robertsipka) [inline jerry-debugger/jerry-client-ws.py]: Simple use `if args:`

File: jerry-debugger/jerry-client-ws.py [hunk 2]
```diff
@@ -169,6 +172,13 @@ def insert_breakpoint(self, args):
...
+            return True
+        else:
```
Comment (robertsipka) [inline jerry-debugger/jerry-client-ws.py]: The else statement is unnecessary, you can just return with false at the end of the function.

File: jerry-debugger/jerry-client-ws.py [hunk 3]
```diff
@@ -469,7 +482,7 @@ def __init__(self, address):
 
         self.idx_format = "I"
 
-        logging.debug("Compressed pointer size: %d" % (self.cp_size))
+        logging.debug("Compressed pointer size: %d", self.cp_size)
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: What is the problem with this?

Comment (polaroi8d) [inline jerry-debugger/jerry-client-ws.py]: There is no problem, but this is the PEP8 logging standard.

Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: OK.

**Pull request merged**